classes/class.tapestry.php

```diff
@@ -134,8 +134,12 @@ public function getNodeIds()
     public function addNode($node)
     {
         $tapestryNode = new TapestryNode($this->postId);
-        $tapestryNode->set($node);
 
+        // If user is author and node is not in review process, grant author edit permissions
+        $userId = wp_get_current_user()->ID;
+        $isAuthor = $userId === $tapestryNode->getAuthor()->id;
```

Comment:
Wouldn't this always be true?

Reply:
Good catch. If a user is adding a node, they're going to be the author. I'll remove it.